ansi/deno.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@cliffy/ansi",
+  "name": "@windmill-labs/cliffy-ansi",
```

Comment:
@rubenfiszel can you revert the package names please? otherwise i cannot merge it.

Reply:
If @rubenfiszel does not have time to tackle this, I could create a PR based off of this one that has the correct package names.

@c4spar Would that be appreciated?